packages/lsp/src/tokens.ts

```diff
@@ -0,0 +1,625 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
Could you add a doc strings for all exported types and interfaces in this file as they will be the primary API used by developers.